### PR TITLE
Configure LKE cluster to publish kubeconfig in plain text

### PR DESCRIPTION
**Pull request by muvaf**
Add sensitive data handler so that base64 encoded kubeconfig appears decoded in the connection details secret.

### CODE REVIEW

File: apis/linode/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```
Comment (gerhard) [inline apis/linode/v1alpha1/zz_generated.deepcopy.go:1]: This file gets modified if I run `make generate`. I am running `go version go1.17.3 linux/amd64`

File: apis/v1alpha1/zz_generated.deepcopy.go [hunk 2]
```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```
Comment (gerhard) [inline apis/v1alpha1/zz_generated.deepcopy.go:1]: Same as above: https://github.com/crossplane-contrib/provider-jet-linode/pull/1/files#r770925611

**Comment by gerhard** [comment]
Apart from these two files which get modified if I run `make generate` everything behaves as expected.
I will merge and then make the change on the main branch.
Thank you!

**Comment by muvaf** [comment]
@gerhard We've seen similar behavior in other providers, it's likely because I have go1.16 and you got 1.17 or higher.

**Pull request merged**